domains/kagurayayoi.json

```
@@ -0,0 +1,10 @@
+{
+    "repo": "https://github.com/fjaxzhy/is-a.dev",
+    "owner": {
+      "username": "KaguraYayoi",
```

username doesnt match github username
```suggestion
      "username": "Fjaxzhy",
```